operating-systems/operating-systems.md

```diff
@@ -1,74 +1,101 @@
 # Operating Systems
 
 ### Projected Time
+
 30-45 minutes
 
 ### Motivation
 
+Whether you interact with it via the command line or through a GUI, the operating system is a crucial and powerful element of any machine. Understanding the fundamentals of what an OS is responsible for and how you can interact with it will help you design more sophisticated and efficient applications.
 
 ### Objectives
+
 **Participants will be able to:**
-- Things
-- More things
-- Even more things
-- Even more things
+
+-   Describe the role of the operating system on a computer or other device
+- 	List several function an operating system performs
```

Comment:
Check out this section in the markdown preview before committing; the bullets are not consistent.

Reply:
thx for noticing! It looked ok in Atom md preview. I adjusted spacing so that there is exactly on space between the hyphen and the first letter of the point.